main.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+require_relative('calendar')
+require_relative('validations')
+require_relative('csv')
+require('date')
+require('colorize')
+require_relative('event')
+
+# class Driver- main class
+class Driver
+  include Validations
+  def initialize
+    @calendar = Calendar.new
+  end
+
+  def run
+    flag = 0
+    while flag != 8
+      show_main_menu
+      flag = validate_integer gets.chomp
+      if !flag.between?(1, 8)
+        puts 'Invalid Choice !'.red
+      elsif flag == 1
+        grid_view
+      elsif flag == 2
+        add_event
+      elsif flag == 3
+        delete_event
+      elsif flag == 4
+        update_event
+      elsif flag == 5
+        month_view
+      elsif flag == 6
+        day_view
+      elsif flag == 7
+        load_from_csv
+      end
+      press_enter
+      system('clear') || system('cls')
+    end
+  end
+
+  def add_event
+    title, venue, date, time = input_event
+    if @calendar.add_event(date, time, title, venue)
+      puts 'Successfully added.'.green
+    else
+      puts 'Event was not added.'.red
+    end
+  end
+
+  def month_view
+    puts 'Please enter date (MM/YYYY)'
+    date = input_date
+    @calendar.month_view(date.month, date.year)
```

Comment:
pass date object directly to month_view method

Reply:
Done